inc/class-cachify.php

```diff
@@ -674,14 +701,45 @@ public static function add_flush_icon( $wp_admin_bar ) {
 			return;
 		}
 
+		echo '
+		<style>
+		    #wp-admin-bar-cachify .animate-pulse:before {
+				animation: cachify-dash-icon-fade 2s infinite;
+				animation-timing-function: linear;
+			}
+
+		    #wp-admin-bar-cachify .animate-success:before {
+				animation: cachify-dash-icon-success 2s;
+				animation-timing-function: linear;
+			}
+
+		    #wp-admin-bar-cachify .animate-fail:before {
+				animation: cachify-dash-icon-fail 2s;
+				animation-timing-function: linear;
+			}
+
+			@keyframes cachify-dash-icon-fade {
+				0%, 100% { opacity: 1 }
+				50% { opacity: 0 }
+			}
+
+			@keyframes cachify-dash-icon-success {
+				from { color: #27b127 }
+			}
+
+			@keyframes cachify-dash-icon-fail {
+				from { color: #ff3b3b }
+			}
+		</style>';
+
 		/* Display the admin icon anytime */
 		echo '<style>#wp-admin-bar-cachify{display:list-item !important} #wp-admin-bar-cachify .ab-icon{margin:0 !important} #wp-admin-bar-cachify .ab-icon:before{content:"\f182";top:2px;margin:0;} #wp-admin-bar-cachify .ab-label{margin:0 5px}</style>';
 
 		/* Add menu item */
 		$wp_admin_bar->add_menu(
 			array(
 				'id'     => 'cachify',
-				'href'   => wp_nonce_url( add_query_arg( '_cachify', 'flush' ), '_cachify__flush_nonce' ), // esc_url in /wp-includes/class-wp-admin-bar.php#L438.
+				'href' => '',
```

Comment:
I would keep the `href` so that it continues to work when the JS fails to load for some reason.